csv-to-json/csvToJson.js

```diff
@@ -1,6 +1,19 @@
+const fs = require('fs');
 
-function csvToJson(...args) {
-  return args;
+function csvToJson(csvFilePath) {
+  const fileContent = fs.readFileSync(csvFilePath, 'utf-8');
+  const newlineSeparatedFileContent = fileContent.split('\n');
+  // console.log(newlineSeparatedFileContent);
+  const keys = newlineSeparatedFileContent[0];
```

Comment:
`keys` isn't used anywhere and I see some code commented below. I guess that this problem is still `WIP` right?

In that case, I suggest that you always add a `//WIP` somewhere in the file, maybe at the top, when you include them in a PR.